### PR TITLE
Raw text parser

### PR DESCRIPTION
**Pull request by nj-kore**
Added a parser that just parses text, and I made this the default parser instead of the pdf one.

### CODE REVIEW

**Comment by nj-kore** [comment]
[Add parser for parsing text files instead of pdfs](https://trello.com/c/0zm4n8yj/13-add-parser-for-parsing-text-files-instead-of-pdfs)

File: scripts/prepdocslib/contentparsers.py [hunk 1]
```diff
@@ -27,17 +33,28 @@ def __init__(self, page_num: int, offset: int, text: str):
...
 
     async def parse(self, content: IO) -> AsyncGenerator[Page, None]:
         if False:
             yield
 
 
-class LocalPdfParser(PdfParser):
+class TextParser(ContentParser):
+    """
+    Parser that parses text which is in byte format.
```
Comment (niklaspaasonen) [inline scripts/prepdocslib/contentparsers.py:48]: I guess its not in `byte format` :)

**Review by niklaspaasonen — APPROVED** [review]
Sweeeeet

File: scripts/prepdocslib/contentparsers.py (hunk 1 above)
Comment (nj-kore) [reply scripts/prepdocslib/contentparsers.py:48]: Well, the file is expected to be opened in bytes format in this case, so when you read the contents of the file, i.e, the text, it is also in byte format, and you have to decode to string.

Comment (niklaspaasonen) [reply scripts/prepdocslib/contentparsers.py:48]: Åh!

**Pull request merged**